bin/resources/GameIndex.yaml

```diff
@@ -29305,7 +29418,7 @@ SLPM-66719:
   name: "Tenka-bito [Sega the Best]"
   region: "NTSC-J"
 SLPM-66720:
-  name: "Guilty Gear XX #Slash [Sega the Best]"
+  name: "Guilty Gear Xx #Slash [Sega the Best]"
```

Comment:
Where did you see that it was Xx and not XX? Everywehre I see it online it's XX.